docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+### Standardized Services
+
+The IOTA Identity framework also standardized certain `services` that are embedded in the DID Document. It is RECOMMENDED to implement these when implementing the `did:iota` method. 
+
+Currently standardized `services`:
+* Nothing yet. 
+
+## Security Considerations
+
+The `did:iota` method is implemented on the IOTA Tangle, a public permissionless and feeless Distributed Ledger Technology (DLT), making it resistant against almost all censorship attack vectors. Up until the `Coordicide` update for the IOTA network, a reliability on the coordinator exists for resolving ordering conflicts. This has a minor censorship possibility, that, in the worst case, can prevent ordering conflicts from being resolving in a DID. However, these can only be published by the possessor of the private key and therefore does not provide a usable attack vector. Lastly, a node may decide to censor DID messages locally, however a user can easily use another node. 
+
+Since DID messages are always to be signed and the 'chain of custody' of the signing key can be traced throughout the identity lifespan, replay, message insertion, deletion, modification and man-in-the-middle attacks are prevented.
+
+### Stateless Identities
+
+Unlike for-purpose Blockchains or Smart contract based DID methods, the IOTA tangle does not track and store the state of DID Document. This prevents dusting attacks against the nodes. However, the client that resolves the DID Document has the responsibility to always validate the entire history of the DID Document as to validate the 'chain of custody' of the signing keys. If a client does not do this, identity hijacking is easily possible. The IOTA Identity framework, which implements the `did:iota` method, provides an easy to use client-side validation implementation. 
+
+### Snapshotting
+
+IOTA allows feeless data and value messages. As such it has a large history of messages combining to several TBs at the time of writing. Most nodes 'snapshot' (forget/delete) older transactions, while they keep the UTXOs stored in the ledger. The snapshot settings are local, therefore all nodes may store a different length of history. As such, older DID messages would not be avaliable at every node. Since the entire history of the DID Document is required for client-side validation, this may become problematic. It is currently recommended to either use your own node, that does not snapshot or use the [Chronicle Permanode](https://github.com/iotaledger/chronicle.rs), which keeps the entire history of the Tangle. This is not a longterm scalable solution, which is why other methods for either keeping state or selective storage is considered. 
```

Comment:
As such it -> As such, it
combining -> amounting
line 4: own node, that does -> own node that does (no comma)
snapshot or use -> snapshot, or use (comma)
storage is considered -> storage are being researched